bokehjs/src/lib/core/layout/layoutable.ts

```diff
@@ -37,6 +37,12 @@ export abstract class Layoutable {
   }
 
   private _dirty: boolean = false
+  set dirty(dirty: boolean) {
+    this._dirty = dirty
+  }
+  get dirty(): boolean {
+    return this._dirty
+  }
```

Comment:
Why not just public `dirty: boolean` at this point?

Reply:
I didn't want to make other changes and this is a temporary workaround anyway. I will revise this, i.e. figure out how to properly detect if re-layout is really needed, as part of the ongoing work on canvas layouts.